public/app/features/datasources/components/DataSourcesList.tsx

```diff
@@ -83,6 +85,21 @@ export function DataSourcesListView({ dataSources, dataSourcesCount, isLoading,
                     dataSource.isDefault && <Tag key="default-tag" name={'default'} colorIndex={1} />,
                   ]}
                 </Card.Meta>
+                <Card.Tags>
+                  <LinkButton icon="apps" fill="outline" variant="secondary" href="/dashboard/new">
+                    Build a Dashboard
+                  </LinkButton>
+                  <LinkButton
+                    icon="compass"
+                    fill="outline"
+                    variant="secondary"
+                    className={styles.button}
+                    href={constructDataSourceExploreUrl(dataSource)}
+                    disabled={!canExploreDataSources}
```

Comment:
I'm wondering if we can hide this link (similar to my previous comment) if the user doesn't have the necessary permissions?

Reply:
Done (Levi also pointed this out :D)